backend/src/main/java/de/grimsi/gameyfin/mapper/GameMapper.java

```diff
@@ -76,13 +76,14 @@ public AutocompleteSuggestionDto toAutocompleteSuggestionDto(Igdb.Game game) {
                 .slug(game.getSlug())
                 .title(game.getName())
                 .releaseDate(ProtobufUtil.toInstant(game.getFirstReleaseDate()))
+                .platforms(game.getPlatformsList().stream().map(Igdb.Platform::getName).collect(toList()))
```

Comment:
Since were using Java 18, you can just call `.toList()` instead of `.collect(Collectors.toList())`

Reply:
Good point!